src/features/click-to-load.js

```diff
@@ -22,6 +22,7 @@ const titleID = 'DuckDuckGoPrivacyEssentialsCTLElementTitle'
 // @see {getConfig}
 let config = null
 let sharedStrings = null
+let styles = getStyles()
```

Comment:
Do we ever use this before `init`?

Reply:
No, but this prevents a typescript error...

Reply:
Can we change the type to make it optional? I think that reduces the risk of unexpected side effects